skimage/segmentation/manual_segmentation.py

```diff
@@ -0,0 +1,274 @@
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+from matplotlib.collections import PatchCollection
+from ..draw import polygon
+
+
+def _mask_from_verts(verts, shape):
+    mask = np.zeros(shape, dtype=bool)
+    pr = [y for x, y in verts]
+    pc = [x for x, y in verts]
+    rr, cc = polygon(pr, pc, shape)
+    mask[rr, cc] = 1
+    return mask
+
+
+def manual(image, alpha=0.4, overlap="merge", mouse=[1, 2, 3]):
+    """Return a binary image based on the selections made with mouse clicks.
+
+    Parameters
+    ----------
+    image : (M, N[, 3]) array
+        Grayscale or RGB image.
+
+    alpha : float, optional
+        Transparancy value for polygons draw over the segments.
+
+    overlap : string, optional
+        Determines the behaviour of mask generation with respect to overlapping
+        regions of polygons.
+
+        * if "merge" : overlapping selections will be merged as a single
+            object.
+        * if "overwrite" : The last polygon overwrites the regions where
+            it intersects with previously drawn polygon.
+        * if "seperate" : overlapping selections will be returned as
+            an array of masks.
+
+    mouse : list of int, optional
+        Customize the mouse click behaviour when selecting vertices with
```

Comment:
I can decode this parameter description with some thinking, but it can be clarified.

Something like:

```
mouse_buttons : 3-tuple of int, optional
    Set the mouse behavior for the left, middle, and right buttons respectively.
    The default setting is `[1, 2, 3]`, where:

    * `1` : ...

    To use ...
```